# Software 

(1) Koster, J.; Rahmann, S. Snakemake--a Scalable Bioinformatics Workflow Engine. Bioinformatics 2012, 28 (19), 2520–2522. https://doi.org/10.1093/bioinformatics/bts480.

(2) Nurk, S.; Meleshko, D.; Korobeynikov, A.; Pevzner, P. A. MetaSPAdes: A New Versatile Metagenomic Assembler. Genome Res 2017, 27 (5), 824–834. https://doi.org/10.1101/gr.213959.116.

(3) Kang, D. D.; Li, F.; Kirton, E.; Thomas, A.; Egan, R.; An, H.; Wang, Z. MetaBAT 2: An Adaptive Binning Algorithm for Robust and Efficient Genome Reconstruction from Metagenome Assemblies. PeerJ 2019, 7. https://doi.org/10.7717/peerj.7359.

(4) Wu, Y.-W.; Simmons, B. A.; Singer, S. W. MaxBin 2.0: An Automated Binning Algorithm to Recover Genomes from Multiple Metagenomic Datasets. Bioinformatics 2016, 32 (4), 605–607. https://doi.org/10.1093/bioinformatics/btv638.

(5) Chaumeil, P.-A.; Mussig, A. J.; Hugenholtz, P.; Parks, D. H. GTDB-Tk: A Toolkit to Classify Genomes with the Genome Taxonomy Database. Bioinformatics 2019, btz848. https://doi.org/10.1093/bioinformatics/btz848.

(6) Chen, S.; Zhou, Y.; Chen, Y.; Gu, J. Fastp: An Ultra-Fast All-in-One FASTQ Preprocessor. Bioinformatics 2018, 34 (17), i884–i890. https://doi.org/10.1093/bioinformatics/bty560.

(7) Li, H.; Durbin, R. Fast and Accurate Short Read Alignment with Burrows-Wheeler Transform. Bioinformatics 2009, 25 (14), 1754–1760. https://doi.org/10.1093/bioinformatics/btp324.

(8) Parks, D. H.; Imelfort, M.; Skennerton, C. T.; Hugenholtz, P.; Tyson, G. W. CheckM: Assessing the Quality of Microbial Genomes Recovered from Isolates, Single Cells, and Metagenomes. Genome Res 2015, 25 (7), 1043–1055. https://doi.org/10.1101/gr.186072.114.

(9) Alneberg, J.; Bjarnason, B. S.; de Bruijn, I.; Schirmer, M.; Quick, J.; Ijaz, U. Z.; Lahti, L.; Loman, N. J.; Andersson, A. F.; Quince, C. Binning Metagenomic Contigs by Coverage and Composition. Nature Methods 2014, 11 (11), 1144–1146. https://doi.org/10.1038/nmeth.3103.

(10) DeMaere, M. Z.; Darling, A. E. Bin3C: Exploiting Hi-C Sequencing Data to Accurately Resolve Metagenome-Assembled Genomes. Genome Biology 2019, 20 (1), 46. https://doi.org/10.1186/s13059-019-1643-1.


# *Nannochloropsis Oceanica* Reference Genome

(11) Guo, L.; Liang, S.; Zhang, Z.; Liu, H.; Wang, S.; Pan, K.; Xu, J.; Ren, X.; Pei, S.; Yang, G. Genome Assembly of Nannochloropsis Oceanica Provides Evidence of Host Nucleus Overthrow by the Symbiont Nucleus during Speciation. Commun Biol 2019, 2 (1), 249. https://doi.org/10.1038/s42003-019-0500-9.


# *Bdellovibrionota​* Annotation

(12) Lee, P. A.; Martinez, K. J. L.; Letcher, P. M.; Corcoran, A. A.; Ryan, R. A. A Novel Predatory Bacterium Infecting the Eukaryotic Alga Nannochloropsis. Algal Research 2018, 32, 314–320. https://doi.org/10.1016/j.algal.2018.04.003.
